src/txn/txn_rollback_to_stable.c

```diff
@@ -1027,14 +1039,16 @@ __rollback_to_stable_btree_apply(WT_SESSION_IMPL *session)
          */
         if (S2BT(session)->modified || max_durable_ts > rollback_timestamp || !durable_ts_found) {
             __wt_verbose(session, WT_VERB_RTS,
-              "%s: file rolled back with durable timestamp: %s, or when tree is modified: %s or "
+              "File rolled back with durable timestamp: %s, or when tree is modified: %s or "
```

Comment:
I prefer to use tree instead of file to maintain abstraction and consistency.

Reply:
Fixed.